integration_tests/test_cargo_audit.py

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+import subprocess 
+
+def test_cargo_audit():
+    cmd = 'cargo-audit'
```

Comment:
Here, we can also pass `-q` flag since we're not interested in the output if the test is passing.

Reply:
you should refresh the page this file should have been deleted

Reply:
It applies to the buildkite fast path too :D.